### PR TITLE
Details style suggestions

### PR DESCRIPTION
**Pull request by eatonphil**

### CODE REVIEW

**Comment by eatonphil** [comment]
This has some changes I'd like to see after #129 gets merged.

**Comment by eatonphil** [comment]
screen shot 2016-06-15 at 1 56 42 pm

**Comment by na3d** [comment]
Is the screen the same?

**Comment by eatonphil** [comment]
The screen?

**Comment by eatonphil** [comment]
Differences you may or may not be able to not in the screenshot:

* Decreased margins around the content of the main card and summary/access section.
* Label colors are #333
* Input and button font and height are standardized

That's all I can think of.

File: scss/manager.scss [hunk 1]
```diff
@@ -285,13 +293,43 @@ nav {
...
+}
+
+.input-group {
+    input {
+        margin: 0;
+        border: $linode-border;
+        border-radius: 0;
+        &[readonly] {
+            background-color: $linode-white;
+            width: 12.5rem;
```
Comment (na3d) [inline scss/manager.scss]: Would it be worthwhile to turn this into a variable?

**Comment by SirCmpwn** [comment]
:+1:

File: scss/manager.scss (hunk 1 above)
Comment (na3d) [inline scss/manager.scss]: Never mind, we can do it later if we run into an instance that needs it.

**Comment by eatonphil** [comment]
Thanks!

**Pull request merged**